crates/biome_js_analyze/src/lint/nursery/no_import_cycles.rs

```diff
@@ -25,6 +25,11 @@ declare_lint_rule! {
     /// only go in a single direction, i.e. they don't point "back" to the
     /// importing file.
     ///
+    /// If a cycle is contained to a single file, i.e. a file imports from
+    /// itself, no warning is issued. This allows for encapsulation of
+    /// functions/variables into a namespace instead of using a static class
+    /// (triggers [no-static-only-class](https://biomejs.dev/linter/rules/no-static-only-class)).
```

Comment:
```suggestion
    /// However, files that import themselves are allowed, and the rule won't trigger for these use cases. 
    /// This allows for encapsulation of
    /// functions/variables into a namespace instead of using a static class
    /// (triggers [no-static-only-class](https://biomejs.dev/linter/rules/no-static-only-class)).
```